src/components/structures/FilePanel.tsx

```diff
@@ -312,4 +311,6 @@ class FilePanel extends React.Component<IProps, IState> {
     }
 }
 
-export default FilePanel;
+export default forwardRef<FilePanel, Omit<IProps, "context">>((props, ref) => (
```

Comment:
Likewise, a word of documentation on this would be useful.

```suggestion
/** Wraps a `FilePanel`, populating its `context` property from the current {@link RoomContext}. */
export default forwardRef<FilePanel, Omit<IProps, "context">>((props, ref) => (
```

Likewise throughout.